src/sandbox/mod.rs

```diff
@@ -207,6 +214,9 @@ pub mod tests {
         assert!(sandbox_debug.contains(sandbox.data.id()));
         assert!(sandbox_debug.contains(&sandbox.data.attempt().to_string()));
         assert!(sandbox_debug.contains(sandbox.data.hostname()));
+
+        let log_dir = sandbox.data.log_directory().to_str().unwrap();
```

Comment:
There is no unwrap necessary here:
```suggestion
        let log_dir = sandbox.data.log_directory().display().to_string();
```

Reply:
Ah, right. Thanks for the suggestion, @saschagrunert. I've just pushed the update.